Core/Core/Kits/Attributes.cs

```diff
@@ -8,17 +8,17 @@ public sealed class SchemaInfo : Attribute
   public SchemaInfo(string name, string description)
     : this(name, description, null, null) { }
 
-  public SchemaInfo(string name, string description, string category, string subcategory)
+  public SchemaInfo(string name, string description, string? category, string? subcategory)
```

Comment:
Not sure about this, all SchemaInfo's should have a category set so the component generator will pick those up.

I'll double check this but if some SchemaInfo does not have category or subcategory i'd consider that a problem :)

Reply:
Happy to adjust if necessary, The constructor above was passing nulls through so I simply typed what I saw.
If unsure, I can simply revert this change, and keep this class null oblivious

Reply:
Oh you are right... 🤦🏼‍♂️ well i'm not sure we should be allowing this. I'll check the usage of this on our object side and get back to you with this

Reply:
There are only 3 usages of this constructor in the entire object model, so I say we add the missing category/subcategory to those and remove (or flag as obsolete) that constructor.

I'd be happier if it were removed to be honest, even if it's a breaking change.

Reply:
Shall we address this in a seperate PR? I can keep this class null oblivious for now.
Then in a separate PR, we can `Obsolete(true)` the ctor,  resolve our own usages, and remove the `nullable disable` (without making any props nullable)

Reply:
yes lets do that to prevent polluting this PR with slightly tangential changes.

Resolving this conversation in favour of https://spockle.atlassian.net/browse/CNX-8542